docs/book/src/multiversion-tutorial/testdata/project/PROJECT

```diff
@@ -18,7 +18,11 @@ resources:
   path: tutorial.kubebuilder.io/project/api/v1
   version: v1
   webhooks:
+    conversion: true
     defaulting: true
+    spoke:
+    - v2
+    - v2
```

Comment:
@varshaprasad96 

Would it make sense to have Many spokes?
If I want to convert my V1 to V2
Why would I want to convert my V1 to V2 and v3 at the same time?